lib/active_record/connection_adapters/crate_adapter.rb

```diff
@@ -131,7 +131,7 @@ def connect
 
       def columns(table_name) #:nodoc:
         cols = @connection.table_structure(table_name).map do |field|
-          name = dotted_name(field[2])
+          name = dotted_name(field[12])
```

Comment:
Thank you for confirming this update, needed for more recent versions of CrateDB. In order to follow up on this within a subsequent iteration, I've just created #24.